pkg/volume/cephfs/cephfs.go

```diff
@@ -262,7 +268,7 @@ func (cephfsVolume *cephfs) execMount(mountpoint string) error {
 	for i = 0; i < l-1; i++ {
 		src += hosts[i] + ","
 	}
-	src += hosts[i] + ":/"
+	src += hosts[i] + ":" + cephfsVolume.subdir
```

Comment:
Fair enough. I will address this and your other comments later today. Thanks for your time.